qt/python/mantidqt/widgets/sliceviewer/view.py

```diff
@@ -315,7 +315,7 @@ def clear_image(self):
             if self.line_plots:
                 self.delete_line_plot_lines()
             self.image_info_widget.updateTable(DBLMAX, DBLMAX, DBLMAX)
-            self.image.remove()
```

Comment:
Just out of curiosity can I ask why the `.remove()` call is no longer sufficient?